.github/workflows/unittest.yml

```diff
@@ -0,0 +1,34 @@
+name: unittest
+
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    services:
+      redis:
+        image: redis
+        options: >-
+          --health-cmd "redis-cli ping"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 6379:6379
+
+    steps:
+      - uses: actions/checkout@v2
+     
+      - name: Setup Python
+        uses: actions/setup-python@v2.2.1
+        with:
+          python-version: '3.x'
+      
+      - name: Install Python Requirements
+        run: pip install -r requirements.txt
+
+      - name: Launch Tests
+        run: pytest doku/tests
```

Comment:
I am surprised that this works. Is a database not necessary? Or does this automatically fall back to use SQLite? I don't remember.
If it's not much work, I'd prefer to actually use a real database server and use the production config if possible. This way we can also test the configuration and setup as used in production.